src/lfo.cpp

```diff
@@ -40,6 +40,25 @@ int32_t LFO::render(int numSamples, int waveType, uint32_t phaseIncrement) {
 	case LFO_TYPE_TRIANGLE:
 		value = getTriangle(phase);
 		break;
+
+	case LFO_TYPE_SAH:
+		if (phase == 0) {
+			value = CONG;
+			holdValue = value;
+		}
+		else if (phase + phaseIncrement * numSamples < phase) holdValue = CONG;
+		else value = holdValue;
+		break;
+
+	case LFO_TYPE_RWALK:
+        uint32_t range = 4294967295u / 20;
+		if (phase == 0) {
+			value = holdValue + (range / 2) - CONG % range;
+			holdValue = value;
+		}
+		else if (phase + phaseIncrement * numSamples < phase) holdValue += (range / 2) - CONG % range;
```

Comment:
This can drift quite far from the midpoint (I had it mapped to filter and it totally closed it for a long time)

Suggest weighting the walk value so that it tends to come back towards range

Reply:
That makes sense. I am just testing a function that adds more bias for walking back the more the value deviates from the origin.

Reply:
Let me know when this is ready to go into community branch.